src/types.ts

```diff
@@ -64,6 +68,13 @@ export type RatelimitResponse = {
    * ```
    */
   pending: Promise<unknown>;
+
+  reason?: ResponseType;
+
+  /**
+   * The value which was in the deny list if reason: "denied"
+   */
+  _deniedValue?: string
```

Comment:
Is there any reason why this is underscored? I believe it is because this is an internal value, but I'd omit the underscore to maintain the convention.

Reply:
I meant that it's for internal use, but it's okay to remove the underscore